src/docs/asciidoc/reference/concepts/zetapush-npm-modules.adoc

```diff
@@ -1,2 +1,69 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+ZetaPush is a *monorepo* that contains several *npm modules*. Each of them have a specific role.
+
+== @zetapush/cli
+
+Contains the *Command Line Interface* of ZetaPush.
+This is thanks to this module that you can launch `zeta run` or `zeta push`. +
```

Comment:
This is thanks to this module that you can launch `zeta run` or `zeta push`. -> You can launch `zeta run` or `zeta push` thanks to this module.